### PR TITLE
Add share and thank you

### PR DESCRIPTION
**Pull request by alicoding**

### CODE REVIEW

**Comment by alicoding** [comment]
@ScottDowne R?

File: public/index.css [hunk 1]
```diff
@@ -582,39 +582,46 @@
...
+}
+.share-page #facebook:hover {
+  background-color: #406db4;
+}
+.share-page #twitter {
+  background-color: #5ECDF3;
+}
+.share-page #twitter:hover {
+  background-color: #46c6f1;
+}
```
Comment (ScottDowne) [inline public/index.css]: Can you change all this indentation back?

Comment (ScottDowne) [inline public/index.css]: If this was intended, R+ otherwise add some whitespace then land.

Comment (alicoding) [inline public/index.css]: it wasn't indentation fixes actually. It was class name changes and add namespace for id/class

**Pull request merged**